src/__tests__/EditorPane.test.tsx

```
@@ -1,28 +1,143 @@
 }));
 
 import { EditorPane } from "../components/shell/EditorPane";
 import { renderWithTheme } from "./test-utils";
+import { ThemeProvider } from "../lib/ThemeContext";
+
+// jsdom does not implement scrollIntoView. Define it as a noop on
+// HTMLElement.prototype so the anchor effect can call it without
+// throwing. Each test spies on the instance method to assert calls.
+if (
+  typeof HTMLElement !== "undefined" &&
+  !HTMLElement.prototype.hasOwnProperty("scrollIntoView")
```

ESLint

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

(no-prototype-builtins)